### PR TITLE
detect-dce-opnum/v2: add test

### PR DESCRIPTION
**Pull request by TheKharleeci**
Adds test for DetectDceOpnumTestParse01 - 02

Previous PR: #700

Link to redmine ticket: https://redmine.openinfosecfoundation.org/issues/4911

Corresponding Suricata PR: OISF/suricata#6910

### CODE REVIEW

File: tests/detect-dce-opnum/test.rules [hunk 1]
```diff
@@ -0,0 +1 @@
+alert tcp any any -> any any (msg:"DCERPC"; dce_opnum:4; sid:1;)
```
Comment (inashivb) [inline tests/detect-dce-opnum/test.rules:1]: The new notation for the keyword is `dce.opnum`. Sorry for not catching this earlier. Please use this. Most keywords with `_` in the name have been changed to this pattern. Latest docs will show you the correct latest usage.

File: tests/detect-dce-opnum/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Description
+
+Tests the dce_opnum keyword
+
+# PCAP
+
+The pcap comes from dcerpc-dce-opnum Suricata-Verify folder.
```
Comment (inashivb) [inline tests/detect-dce-opnum/README.md:7]: This makes me realize we probably do not need a new test then, could you please update the existing `dcerpc-dce-opnum` test with this rule and corresponding `test.yaml`?

File: tests/detect-dce-opnum/test.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+requires:
+  min-version: 6
```
Comment (inashivb) [inline tests/detect-dce-opnum/test.yaml:2]: Previous comment about this version check only applied to `dcerpc` logging so I meant this min-version check should go in the filter for `dcerpc` logs only. Here, it applies to the `alert` check too.

**Review by inashivb — CHANGES_REQUESTED** [review]

**Comment by TheKharleeci** [comment]
Followed By: #704

**Pull request closed without merging**